base/docs/helpdb/Base.jl

```diff
@@ -423,24 +407,19 @@ Show every part of the representation of a value.
 dump
 
 """
-    display(x)
-    display(d::Display, x)
-    display(mime, x)
-    display(d::Display, mime, x)
+    isinteractive() -> Bool
+
+Determine whether Julia is running an interactive session.
+"""
+isinteractive
```

Comment:
I'm really confused about why github is showing a merge conflict. If I look at the files changed tab, it matches what I have on my desktop, which has `isinteractive` still in helpdb...